### DISCUSSION TITLE
Switch based on media RTSP source?

### DISCUSSION THREAD

**Discussion by jaredcat** [WarmUpTill/SceneSwitcher#780] [Q&A]
Is it possible to switch to a source based on if a media RTSP (or possibly NDI) source is receiving data?

Example: I want to add a source for a RSTP stream from my phone. When I start streaming from my phone, automatically switch to the scenes.

**Comment by WarmUpTill** [comment]
Not directly I think, but you could potentially use the [Video condition](https://github.com/WarmUpTill/SceneSwitcher/wiki/Video-condition) to check if the given video source has valid output / has changed.

Let me know if you have questions and sorry about the delayed response.
